wirey.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I think this would be useful in the local development section of the readme.

Reply:
I removed this file, added to gitignore and added a new entry on the README.md file